static/js/components/tiles/chart_footer.tsx

```diff
@@ -26,17 +26,28 @@ import {
   GA_PARAM_TILE_TYPE,
   triggerGAEvent,
 } from "../../shared/ga_events";
+import { ChartActions } from "./chart_action_icons";
+import { ChartDownloadSpec } from "./modal/chart_download";
 
 // Number of characters in footnote to show before "show more"
 const FOOTNOTE_CHAR_LIMIT = 150;
 
 interface ChartFooterPropType {
+  // Callback to run after "download" is clicked
   handleEmbed?: () => void;
+  // Id of the chart this footer attaches to
+  chartId: string;
+  // Containing element to append tooltips/modals to
+  container?: HTMLElement;
   // Link to explore more. Only show explore button if this object is non-empty.
   exploreLink?: { displayText: string; url: string };
   children?: React.ReactNode;
   // Text to show above buttons
   footnote?: string;
+  // Whether to use new chart action icons
+  useChartActionIcons?: boolean;
```

Comment:
Can we completely encapsulate whether the dev flag is turned on or not within the actual chart footer component? This way the upstream components (block.tsx, vis tools, etc) don't need to know that there's a dev flag set or not